contracts/core/RebalancingSetToken.sol

```diff
@@ -241,8 +256,36 @@ contract RebalancingSetToken is
         // Must be in Rebalance state to call settlement
         require(rebalanceState == State.Rebalance);
 
+        // Make sure all currentSets have been rebalanced
```

Comment:
This comment refers to the name of the function.

The method/function name `settlement` seems more like a process than a verb. Can we rename the function to `settleRebalance` or `resolveReblance` or something better?

Reply:
Sure I can change...don't feel strongly either way

Reply:
Yeah, the other functions are verbs. Let's rename for consistency.